include/Sales.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef SALES_HPP
+#define SALES_HPP
+
+#include <string>
+#include <vector>
+#include "Item.hpp"
+
+class Sale
+{
+friend class SaleList;
+public:
+    /*
+     * Sale constructor that sets all the data for the given sale
+     * Note it is checked at some point that most the numbers are not 0 and strings are not empty.
+     */
+    Sale(const unsigned int, const unsigned int, const std::string&, const unsigned int, const double, const double, const std::string&, const std::string&);
+protected:
+    unsigned int identification;
+    unsigned int sale_number;
+    std::string date;
+    unsigned int amount_sold;
+    double sale_price;
+    double tax;
+    std::string buyer;
+    std::string seller;
+    double total_price;
+};
+
+class SaleList
+{
+public:
+
+    /*
+     * Constructor, will take in the name of the file that the sales List will be working from.
+     */
+    SaleList(const std::string&);
+
+    /*
+     * Adds a sale to the vector by using the Sale class constructor, returns false if an error occurs
+     * These errors include certain numbers being 0 or strings being empty.
+     */
+    bool add_sale(const unsigned int, const unsigned int, const std::string&, const unsigned int, const double, const double, const std::string&, const std::string&);
```

Comment:
For consistency purposes, change these to camelCase since that's what the functions in the other classes are using.  So, change to ```addSale``` and etc for the other functions.